src/focus.service.ts

```diff
@@ -362,46 +332,35 @@ export class FocusService {
       return;
     }
 
-    this.rescroll(next, scrollSpeed, this.root);
-
     const attached = selected && isNodeAttached(selected, this.root);
 
-    this.parents = [];
-
     if (attached) {
       // Find the common ancestor of the next and currently selected element.
       // Remove selected classes in the current subtree, and add selected
       // classes in the other subtree. Trigger focus changes on every
       // element that we touch.
-      selected.blur();
-
       const common = getCommonAncestor(next, selected);
-      selected.classList.remove(cssClass.direct);
```

Comment:
The idea is to have different focus state for keyboard nav vs point&click nav
https://github.com/WatchBeam/ui/blob/81cc92782fdc64c21f42930bd27eb4cb1f774e32/src/lib/button/button.scss#L47